docs/README.md

```diff
@@ -346,6 +346,203 @@ Status code | Explanation
 
 Note, that at the moment HTTP 400 may occur also for 3rd party reasons - eg. bacause Nordea test API does not support refunds. See all provider limitations from [providers tab](/payment-method-providers#refunds).
 
+## Tokenized credit card payments
+
+Checkout provides an API for tokenizing payment cards and issuing payments on those tokenized payment cards.
+
+### Adding (tokenizing) cards
+
+Checkout utilizes its [PaymentHighway](https://www.paymenthighway.io/)-service for tokenizing cards.
```

Comment:
Not sure if mentioning paymenthighway is ok here? Ask Joni what he thinks.

Reply:
Same thing through the document. Make sure paymenthighway is still correct term.

Reply:
Hmm yeah I thought it might be good to clarify this, since the user is directed to a 3rd party service which might be confusing at first for the merchant devs.

But good point, will ask.

Reply:
Removed paymenthighway mentions for now :)

Reply:
Removed paymenthighway mentions 👌